docs/asyncapi.yaml

```diff
@@ -1,7 +1,7 @@
 asyncapi: 2.4.0
 info:
   title: FFC Payment Statement Generator
-  version: '1.0.0'
+  version: '1.11.1'
```

Comment:
This version doesn't correlate with the version of the service at large -- meant to version the document -- we don't actually have a process around this, if you had any ideas... 👀